pages/team.js

```diff
@@ -46,20 +46,26 @@ export default function Team() {
                 img="/team/zach.jpg"
                 name="Zach Latta"
                 teamRole="Founder"
+                handle="zachlatta"
+                link="https://github.com/zachlatta"
                 text="Zach dropped out of high school after his freshman year to work in the technology industry and had over 5 million people using his software by the time he turned 17. He founded Hack Club to build the program he wish he had in high school and has been awarded the Thiel Fellowship and Forbes 30 Under 30 for his work."
                 pronouns="he/him"
               />
               <Bio
                 img="/team/christina.jpg"
                 name="Christina Asquith"
                 teamRole="COO"
+                handle="christina"
+                link="https://christina.cool/"
```

Comment:
Does this link work? It doesn't seem to work for me

Reply:
> Thanks for adding this @bellesea! I added a few comments.
> 
> I'm not sure about the placement of the handle in between the role and pronouns; it feels a bit out of place there. Thoughts on having it in a muted colour & small font size next to the name instead? Or perhaps just the `mention` icon?
> 
> Also, could you run `yarn lint` sometime before this gets merged? Thanks!
> 
> ###### edit: oops I didn't mean to click the "Approve" button; I think it's looking good so far though!

@exu3 i agree about that the placement could be better... a mock up of next to name or just @ icon. maybe we could directly link it to the name?

<img width="1434" alt="Screenshot 2022-05-29 at 4 20 16 PM" src="https://user-images.githubusercontent.com/65808924/170858969-cc6209d9-eb9c-40ea-8b99-e1bd25baebcc.png">

<img width="1435" alt="Screenshot 2022-05-29 at 4 21 27 PM" src="https://user-images.githubusercontent.com/65808924/170859003-5ce54189-c3f3-4099-a550-b0e4fac94a9d.png">

what are your thoughts on these? (also, just ran yarn lint, thanks!)

Reply:
Ooh linking directly to the name might be a good idea

Thoughts on the design below, using the `mention` icon (from https://icons.hackclub.com)?
<img width="435" alt="Screen Shot 2022-05-29 at 01 30 14" src="https://user-images.githubusercontent.com/72365100/170859289-0f3b591d-06fb-49e7-8755-406eb58d20d9.png">

Reply:
ooo i love that idea!

Uploading Screen Recording 2022-05-29 at 4.49.23 PM.mov…

Reply:
@sampoder what do you think about this?

Reply:
I honestly can't see it, but that might just be my eyes. I think it looks a bit off because the @ normally comes at the start

Reply:
But I do prefer the icon over the char

Reply:
This design is hard... it feels like things are starting to get cluttered. My fave I've seen is this one:

![](https://user-images.githubusercontent.com/65808924/170858969-cc6209d9-eb9c-40ea-8b99-e1bd25baebcc.png)

Reply:
yeah i agree the contrast isn't good! what about this? or maybe we can just link the names so we don't need another element...

<img width="1435" alt="Screenshot 2022-05-29 at 8 08 48 PM" src="https://user-images.githubusercontent.com/65808924/170878355-f4bc940f-09fd-49b6-b61c-5e9a2671d69b.png">

Reply:
@sampoder what are your thoughts on the design above^?